src/MagnumPlugins/StbResizeImageConverter/Test/StbResizeImageConverterTest.cpp

```diff
@@ -337,6 +340,43 @@ void StbResizeImageConverterTest::rg16() {
         DebugTools::CompareImage);
 }
 
+void StbResizeImageConverterTest::rg16f() {
+    /* Like rgb8Padded(), but converted to half-float and dropping the B
+       channel */
+    const UnsignedShort input[][2]{
+        {Math::packHalf(Math::unpack<Float, UnsignedByte>(0xff)),
+            Math::packHalf(Math::unpack<Float, UnsignedByte>(0x33))},
+        {Math::packHalf(Math::unpack<Float, UnsignedByte>(0xff)),
+            Math::packHalf(Math::unpack<Float, UnsignedByte>(0x66))},
+        {Math::packHalf(Math::unpack<Float, UnsignedByte>(0x66)),
+            Math::packHalf(Math::unpack<Float, UnsignedByte>(0xff))},
+        {Math::packHalf(Math::unpack<Float, UnsignedByte>(0x99)),
+            Math::packHalf(Math::unpack<Float, UnsignedByte>(0x33))},
+        {Math::packHalf(Math::unpack<Float, UnsignedByte>(0x33)),
+            Math::packHalf(Math::unpack<Float, UnsignedByte>(0x99))},
+        {Math::packHalf(Math::unpack<Float, UnsignedByte>(0xcc)),
+            Math::packHalf(Math::unpack<Float, UnsignedByte>(0xcc))}
+    };
+    const UnsignedShort expected[][2]{
+        /* 0xba4d (0.729412, 0.301961), 0x99c3 (0.6, 0.764706) was in the 8-bit
+           case */
+        {Math::packHalf(0.7305f), Math::packHalf(0.3005f)},
+        {Math::packHalf(0.5981f), Math::packHalf(0.7637f)}
```

Comment:
Could be just

```suggestion
        {0.7305_h, 0.3005_h},
        {0.5981_h, 0.7637_h}
```

(`#include <Magnum/Math/Half.h>`). For the above you could do casts with `Half{...}` instead of `Math::packHalf(...)` to make it slightly shorter, unfortuantely I don't know anything better than that.